lib/menu.js

```diff
@@ -26,6 +28,32 @@ module.exports = function (app) {
               }
             })
           }
+        },
+        {
+          label: 'Export as GeoJSON..',
+          click: function (item, focusedWindow) {
+            dialog.showSaveDialog({
+              title: 'Guardar como..',
+              defaultPath: 'export.geojson',
+              filters: [{name: 'GeoJSON', extensions: ['geojson']}]
+            }, function (filename) {
+              if (!filename) return
+              var bbox = [-Infinity, -Infinity, Infinity, Infinity]
+              var out = fs.createWriteStream(filename)
+              var ex = exportGeoJson(app.osm, {bbox: bbox})
+              pump(ex, out, function (err) {
```

Comment:
I went and wrote a simple progress bar view, then of course realized: an osm-p2p-db query stream's length is *unknown* at stream-time. We could show a dialog that displayed how long we've been waiting or how many objects were processed, but no real indication of how much is left.

I'll leave my progress bar work in another branch on `origin`, should we wish to pick that back up later.